hasher-matcher-actioner/hmalib/tests/test_models.py

```diff
@@ -206,3 +219,15 @@ def test_query_match_record_by_time(self):
         query_record = models.PDQMatchRecord.get_from_time_range(self.table)[0]
 
         assert record == query_record
+
+
+class LabelsTestCase(unittest.TestCase):
+    def test_label_validation(self):
```

Comment:
nit: still needed with lower test?

Reply:
didn't write the test, just consolidating but this might help narrow down if the issue real is serde or not